src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -671,9 +671,11 @@ void ProcessOverride (OverrideInformation overrideInformation)
 
 			var isInstantiated = Annotations.IsInstantiated (method.DeclaringType);
 
+			var isStaticInterfaceOverride = @base.IsStatic && method.IsStatic;
```

Comment:
I think it's worth a comment what's going on here.
What path is a static interface method expected to take through this method - the method is already pretty complex, and it seems we rely on relatively fragile things for the static interface method to make it through and return `true`,